src/Build.UnitTests/BuildEventArgsSerialization_Tests.cs

```diff
@@ -358,6 +360,30 @@ public void RoundtripProjectImportedEventArgs()
                 e => e.ProjectFile);
         }
 
+        [Fact]
+        public void RoundtripTargetSkippedEventArgs()
+        {
+            var args = new TargetSkippedEventArgs(
+                "Message")
+            {
+                BuildEventContext = BuildEventContext.Invalid,
+                ProjectFile = "foo.csproj",
+                ParentTarget = "bar",
+                BuildReason = TargetBuiltReason.DependsOn
+            };
```

Comment:
I've just realized that @panopticoncentral @AndyGerlicher we forgot to serialize the TargetName. I think we'll need a separate PR that bumps the format version again.

Or, @AndyGerlicher you could sneak it in file format version 5 together with your profile changes?

Just need to read and write the TargetName for TargetSkippedEventArgs.